evalml/tests/component_tests/test_stacked_ensemble_classifier.py

```diff
@@ -82,7 +83,7 @@ def test_stacked_ensemble_n_jobs_negative_one(X_y_binary, logistic_regression_bi
     clf.fit(X, y)
     y_pred = clf.predict(X)
     assert len(y_pred) == len(y)
-    assert not np.isnan(y_pred).all()
+    assert not np.isnan(y_pred.to_series()).all()
```

Comment:
I feel like the woodwork structures should have started as a numpy array base class lol

Reply:
Bahaha I can see why! My guess is that they chose to start with a pandas base since they take advantage of a lot of the new nullable pandas types~